test/spec/features/modeling/ReconnectConnectionSpec.js

```diff
@@ -106,8 +108,12 @@ describe('features/modeling - reconnect connection', function() {
 
       it('should execute', inject(function(modeling) {
 
+        // given
         var newWaypoints = [ { x: 110, y: 110 }, { x: 300, y: 300 } ];
 
+        // stub layouter to not interfere with results;
+        sinon.stub(modeling, 'layoutConnection').returns();
```

Comment:
I'd love to do less spying here so we keep our test as integration-testish as possible. If you do not want a certain behavior in your tests you can simply stub out the whole component (cf. https://github.com/bpmn-io/diagram-js/pull/320/commits/11baa5ce9515c4947eabf71b65dc63c9bb4d2f36) how I did it.